CHANGELOG.md

```diff
@@ -5,249 +5,261 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0).
 
+## [0.26.0] - 2022-01-25
+
+### Added
+
+* Categorical eligibility to school meal subsidies.
```

Comment:
my formatter changed all the `-` to `*` - @nikhilwoodruff could we try to figure this out when we meet? vscode also just started indenting my yaml files four spaces instead of two...